src/pages/funds/fund-details/components/fund-details-description-section/fund-details-description/fund-details-description.js

```diff
@@ -79,7 +80,12 @@ class FundDetailsDescription extends PureComponent {
           <div className="fund-details-description__heading">
             {fundDescription.title}
           </div>
-          <Link to={composeManagerDetailsUrl(fundDescription.manager.url)}>
+          <Link
+            to={{
+              pathname: composeManagerDetailsUrl(fundDescription.manager.url),
+              state: `${backPath} / ${fundDescription.title}`
```

Comment:
If we can not click on the grandchild route, I think we should not show that. (state: ` / ${fundDescription.title}`)